src/coreclr/src/pal/src/libunwind/src/CMakeLists.txt

```diff
@@ -333,35 +334,76 @@ SET(libunwind_x86_64_la_SOURCES_x86_64
+        SET(libunwind_elf_la_SOURCES                ${libunwind_elf64_la_SOURCES})
+        list(APPEND libunwind_setjmp_la_SOURCES     x86_64/longjmp.S x86_64/siglongjmp.SA)
+    endif()
 
-add_library(libunwind
-  OBJECT
-  ${libunwind_la_SOURCES}
-  ${libunwind_remote_la_SOURCES}
-  ${libunwind_dwarf_local_la_SOURCES}
-  ${libunwind_dwarf_common_la_SOURCES}
-  ${libunwind_dwarf_generic_la_SOURCES}
-  ${libunwind_elf_la_SOURCES}
-)
+    add_library(libunwind
+      OBJECT
+      ${libunwind_la_SOURCES}
+      ${libunwind_remote_la_SOURCES}
+      ${libunwind_dwarf_local_la_SOURCES}
+      ${libunwind_dwarf_common_la_SOURCES}
+      ${libunwind_dwarf_generic_la_SOURCES}
+      ${libunwind_elf_la_SOURCES}
+    )
+else(CLR_CMAKE_HOST_UNIX)
+    if(CLR_CMAKE_TARGET_ARCH_ARM64)
+        SET(libunwind_la_SOURCES                    ${libunwind_la_SOURCES_aarch64})
+        SET(libunwind_remote_la_SOURCES             ${libunwind_aarch64_la_SOURCES_aarch64})
+        SET(libunwind_elf_la_SOURCES                ${libunwind_elf64_la_SOURCES})
+        list(APPEND libunwind_setjmp_la_SOURCES     aarch64/siglongjmp.S)
+    elseif(CLR_CMAKE_TARGET_ARCH_ARM)
+        SET(libunwind_la_SOURCES                    ${libunwind_la_SOURCES_arm})
+        SET(libunwind_remote_la_SOURCES             ${libunwind_arm_la_SOURCES_arm})
+        SET(libunwind_elf_la_SOURCES                ${libunwind_elf32_la_SOURCES})
+        list(APPEND libunwind_setjmp_la_SOURCES     arm/siglongjmp.S)
+    elseif(CLR_CMAKE_TARGET_ARCH_I386)
+        SET(libunwind_la_SOURCES                    ${libunwind_la_SOURCES_x86} ${libunwind_x86_la_SOURCES_os})
+        SET(libunwind_remote_la_SOURCES             ${libunwind_x86_la_SOURCES_x86})
+        SET(libunwind_elf_la_SOURCES                ${libunwind_elf32_la_SOURCES})
+        list(APPEND libunwind_setjmp_la_SOURCES     x86/longjmp.S x86/siglongjmp.S)
+    elseif(CLR_CMAKE_TARGET_ARCH_AMD64)
+        SET(libunwind_la_SOURCES                    ${libunwind_la_SOURCES_x86_64})
+        SET(libunwind_remote_la_SOURCES             ${libunwind_x86_64_la_SOURCES_x86_64})
+        SET(libunwind_elf_la_SOURCES                ${libunwind_elf64_la_SOURCES})
+        list(APPEND libunwind_setjmp_la_SOURCES     x86_64/longjmp.S x86_64/siglongjmp.SA)
+    endif()
+
+    set_source_files_properties(${CLR_DIR}/src/pal/src/exception/remote-unwind.cpp PROPERTIES COMPILE_FLAGS /TP INCLUDE_DIRECTORIES ${CLR_DIR}/src/inc)
+    set_source_files_properties(${CLR_DIR}/src/pal/src/exception/seh-unwind.cpp PROPERTIES COMPILE_FLAGS /TP INCLUDE_DIRECTORIES ${CLR_DIR}/src/inc)
+    add_library(libunwind_xdac
+      OBJECT
+      ../../exception/remote-unwind.cpp
+      ../../exception/seh-unwind.cpp
+      win/pal-single-threaded.c
+      # ${libunwind_la_SOURCES}  Local...
+      ${libunwind_remote_la_SOURCES}
+        # Commented out above for LOCAL + REMOTE runtime build
```

Comment:
A nit - alignment

Reply:
I had deliberately indented that.  But I can unindent if you prefer.